recipe/meta.yaml

```diff
@@ -27,38 +28,58 @@ outputs:
         - python
         - packaging
         - pandas >=1.4.0
+        - shapely >=2.0.0
+        - numpy >=1.22
+        - pyogrio >=0.7.2
         - pyproj >=3.3.0
-        - shapely >=1.8.0
-        - fiona >=1.8.21
+      run_constrained:
+        # geodatabase access
+        # https://github.com/geopandas/geopandas/blob/v1.0.1/environment-dev.yml#L35
+        - psycopg2 >=2.8.0
+        - psycopg-binary >=3.1.0
+        - sqlalchemy >=1.3
+        - matplotlib >=3.5.0
+        - matplotlib-base >=3.5.0
+        # parquet
+        - pyarrow >=8.0.0
     test:
       imports:
         - geopandas
         - geopandas.io
-        - geopandas.tools
-        - geopandas.datasets
       files:
         - test_data
+      source_files:
+        - geopandas/tests
+        - geopandas/conftest.py
+        - geopandas/io/tests
+        - geopandas/tools/tests
       requires:
         - pip
+        - pytest
+        - pytest-xdist
       commands:
         - pip check
+        # check that pip gets the correct version 
+        - python -c "from importlib.metadata import version; assert(version('{{ name }}')=='{{ version }}')"
+        - pytest -n auto -vv geopandas/tests
+        - pytest -n auto -vv geopandas/io/tests
+        - pytest -n auto -vv geopandas/tools/tests
 
-  - name: geopandas
+  - name: {{ name }}
     requirements:
       host:
         - python
         - pip
-        - setuptools
+        - setuptools >=61.0.0
         - wheel
       run:
         - python
         - {{ pin_subpackage('geopandas-base', exact=True) }}
+        # extras for: plotting
         - folium
         - mapclassify >=2.4.0
```

Comment:
```suggestion
        # https://github.com/geopandas/geopandas/blob/v1.0.1/doc/source/getting_started/install.rst#dependencies
        # in the next version it will be bumped to 2.5.0 and added to pyproject.toml
        - mapclassify >=2.4.0
```